docker/api/container.py

```diff
@@ -834,8 +834,13 @@ def logs(self, container, stdout=True, stderr=True, stream=False,
                 else:
                     if isinstance(since, datetime):
                         params['since'] = utils.datetime_to_timestamp(since)
-                    elif (isinstance(since, int) and since > 0):
-                        params['since'] = since
+                    elif isinstance(since, int):
+                        if since > 0:
+                            params['since'] = since
+                        else:
+                            raise errors.InvalidArgument(
+                                'since value should be greater than 0'
+                            )
                     else:
                         raise errors.InvalidArgument(
                             'since value should be datetime or int, not {}'.
```

Comment:
Instead of adding more conditional branches, I'd rather update this message to say "datetime or positive int`. What do you think?

Reply:
Nice, I'll update it.